presto-hive/src/main/java/com/facebook/presto/hive/HiveMetadata.java

```diff
@@ -212,6 +221,7 @@
     private final String prestoVersion;
     private final HiveStatisticsProvider hiveStatisticsProvider;
     private final int maxPartitions;
+    private final CollectibleStatisticsProvider collectibleStatisticsProvider;
```

Comment:
Let's move this above, to be after `HiveStatisticsProvider`

Reply:
`CollectibleStatisticsProvider` has been removed in favor of `ExtendedHiveMetastore#getSupportedColumnStatistics`.